utils/kata-manager.sh

```diff
@@ -584,6 +594,46 @@ install_nerdctl()
 		done
 
 	info "$project installed\n"
+
+	sudo mkdir -p /opt/cni/bin
+
+	# nerdctl requires cni plugins to be installed in /opt/cni/bin
+	# Copy extracted tarball cni files under /usr/local/libexec
+	sudo cp /usr/local/libexec/cni/* /opt/cni/bin/
```

Comment:
I wonder if we could sym-linke these instead?

Reply:
I was thinking of sym-linking these first, but then decided against it to keeps things same in case of containerd install where the files are copied.